ResearchKit/ActiveTasks/ORKTowerOfHanoiTowerView.m

```diff
@@ -66,7 +66,8 @@ - (instancetype)initWithFrame:(CGRect)frame maximumNumberOfDisks:(NSUInteger)max
 
 - (void)updateConstraints {
     if (_currentConstraints != nil) {
-        [NSLayoutConstraint deactivateConstraints:_currentConstraints];
+        [self removeConstraints:_currentConstraints];
```

Comment:
Done! rdar://22179283

I attached a minimum example Xcode project showing the crash. You can also see it [here](https://github.com/rsanchezsaez/iOS9-Constraint-Crash). I didn't have to include *ResearchKit*, this example reproduces the crash with very little code.

It happens when you call `deactiveConstraints:` with an array that has constraints for a view that was removed.

---